cobalt/black_box_tests/black_box_tests.py

```diff
@@ -112,6 +111,20 @@
 _TESTS_EVERGREEN_END_TO_END = [
     'evergreen_verify_qa_channel_update_test',
 ]
+# These tests need more work
+_FILTERED_TESTS = [
```

Comment:
We might want to put these tests in a platform configuration file instead of here. The reason is that black_box_tests is still running on Linux today. And the filters here will impact those platform running them.
One possible location to put the filters is https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/android/shared/test_filters.py